setup.py

```diff
@@ -8,6 +8,8 @@
 
 deps = [
     'marionette-client == 0.9',
+    'mozinstall == 1.11',
+    'mozlog == 2.10',
```

Comment:
mozlog comes in with marionette. If the version is too low we can probably bump it in the client's requirements.txt